config/locales/en.yml

```diff
@@ -449,6 +449,8 @@ en:
         verify: Make sure the verification file is placed in the folder and click verify.
         note: "Note: this process may take a few minutes to several hours depending on where your site is hosted."
       verify_button: Verify
+    create:
+      duplicate_channel: "%{domain} is already present."
```

Comment:
minor: It might be nice to explain what the user should do like

"Your account has a pending verification for %{domain} and you can continue it %{here}."